apps/courses/src/routes/privacy-policy/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Heading,
+  Text,
+  List,
+  ListItem,
+  Divider,
+  Flex,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Icon,
+} from '@chakra-ui/core';
+import Page from '@openmined/shared/util-page';
+import content from '../../content/privacy-policy';
+import GridContainer from '../../components/GridContainer';
+
+import { Link, animateScroll as scroll } from 'react-scroll';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronUp, faCommentAlt } from '@fortawesome/free-solid-svg-icons';
+import CircledNumber from '../../components/CircledNumber';
+
+const SectionListItem = ({ content, index, onClick, ...props }) => (
+  <ListItem cursor="pointer" {...props}>
+    <Link
+      to={`accordion-button-${content.title
+        .replace(/\s+/g, '-')
+        .toLowerCase()}`}
+      onClick={() => onClick(index)}
+      smooth={true}
+      offset={-150}
+      duration={500}
+    >
+      <Flex alignItems="center">
+        <CircledNumber
+          backgroundColor="gray.800"
+          mr="2rem"
+          color="white"
+          size="2rem"
+          text={index + 1}
+        />
+        <Text color="gray.700">{content.title}</Text>
+      </Flex>
+    </Link>
+  </ListItem>
+);
+
+export default () => {
+  const [sectionIndexes, setSectionIndexes] = useState([0]);
+
+  const { title, last_updated } = content.heading;
+  const { sections } = content;
+  const { footer } = content.sidebar;
+
+  const SIDEBAR_WIDTH = 280;
```

Comment:
I've seen us using this number quite a bit. You don't need to do anything here, but I feel we should standardize this somewhere.